scripts/unit.js

```diff
@@ -1,8 +1,9 @@
 'use strict'
 
 const { exec } = require('node:child_process')
+const { argv } = require('node:process')
 
-const child = exec('npm run unit:with-modules')
+const child = exec(argv[2] + ' run unit:with-modules')
```

Comment:
Can you please add `npm` as a default?

You'd need to add `bun` to CI as well.